spel/scripts/amigen7-build.sh

```diff
@@ -61,9 +88,14 @@ then
 fi
 DEFAULTREPOS+=(epel)
 
-if [[ -z "${CUSTOMREPONAME}" ]]
+# Custom repo-def RPMs to install
+if [[ ${AMIGENREPOS} == "UNDEF" ]]
 then
-    CUSTOMREPONAME=$(IFS=,; echo "${DEFAULTREPOS[*]}")
+   err_exit "Installing no custom repo-config RPMs" NONE
+   AMIGENREPOS=$(IFS=,; echo "${DEFAULTREPOS[*]}")
+#else
```

Comment:
This else is actually unnecessary. It is handled by the logic around CUSTOMREPORPM.